example/flutter_example/assets/example.toml

```diff
@@ -0,0 +1,16 @@
+str = "The quick brown fox jumps over the lazy dog."
+
+literal_str = 'C:\Users\nodejs\templates'
+
+multiline_str = """\
+The quick brown \
+fox jumps over \
+the lazy dog.\
+"""
+
+literal_multiline_str = '''
+The first newline is
+trimmed in raw strings.
+   All other whitespace
+   is preserved.
+'''
```

Comment:
I think a more minimal example like 

```toml
str = 'Hello, World!'
```

would be sufficient here. The user is probably more interested in how to load the configuration file when using Flutter.